anthos-bm-gcp-terraform/versions.tf

```diff
@@ -15,6 +15,16 @@
  */
 
 terraform {
+  provider_meta "google" {
+    # the following versions are only bumped on releases; thus the module
+    # versions below might be different from what's used in the terraform
+    # scripts on the master branch
+    module_name           = "anthos-samples/terraform/anthos-bm-terraform:gce/v0.1.0"
+    gcloud                = "terraform-google-modules/gcloud/google/v2.1.0"
+    gcp_apis              = "terraform-google-modules/project-factory/google//modules/project_services/v10.3.2"
+    gcp_service_accounts  = "terraform-google-modules/service-accounts/google/v~> 4.0"
+    gcp_compute_instances = "terraform-google-modules/vm/google//modules/compute_instance/v~> 6.3.0"
```

Comment:
AFAIK only module_name is supported by the provider in `provider_meta`

Reply:
@bharathkkb - that is correct. However, I would want the versions of the other modules used inside also to be maintained in one place here and update with every release.

The current `terraform-module` releaser wouldn't achieve the above for us. So I have just kept the versions here for now and added [an issue](https://github.com/GoogleCloudPlatform/anthos-samples/issues/45) to work on a new release-type for the release bot to support this as well.

Reply:
Got it, commented in the issue.